dstep/Configuration.d

```diff
@@ -67,6 +69,14 @@ struct Configuration
     /// space after function name
     @("space-after-function-name", "Put a space after a function name [default].")
     bool spaceAfterFunctionName = true;
+
+    /// do not translate bodies of following structs and unions
+    @("skipdef", "Keep only the signature of <symbol>.")
```

Comment:
Maybe we should add short options (single letter) as well. If someone have many symbols to skip the CLI can become very long...